### PR TITLE
Update ember-cli 1.13.7 to 2.5.1

### PR DESCRIPTION
**Pull request by mixonic**
Just jumping through the hoops to upgrade ember-cli. Addressed a local build issue (which I presume was related to npm3 incompatibility).

### CODE REVIEW

File: tests/dummy/config/environment.js [hunk 1]
```diff
@@ -19,10 +19,6 @@ module.exports = function(environment) {
     }
   };
 
-  if (environment === 'production') {
-    ENV.baseURL = '/ember-wormhole'; // for gh-pages live demo
-  }
-
```
Comment (chrislopresto) [inline tests/dummy/config/environment.js]: We will still want this

**Comment by chrislopresto** [comment]
Thanks for doing this!

File: tests/dummy/config/environment.js (hunk 1 above)
Comment (mixonic) [inline tests/dummy/config/environment.js]: whoooops :-p will fix

**Comment by mixonic** [comment]
@chrislopresto should be g2g!

**Pull request merged**